scripts/Draconic-Evolution.zs

```diff
@@ -725,16 +725,10 @@ mods.avaritia.ExtremeCrafting.addShaped(<DraconicEvolution:draconium:1>, [
 [null, null, null, null, null, null, null, null, null]]);
 
 // --- Draconic Chest
-mods.avaritia.ExtremeCrafting.addShaped(<DraconicEvolution:draconiumChest>, [
-[null, null, null, null, null, null, null, null, null],
-[null, null, null, null, null, null, null, null, null],
-[null, null, <ore:screwDraconium>, <ore:plateDoubleDraconium>, <ore:plateDoubleDraconium>, <ore:plateDoubleDraconium>, <ore:screwDraconium>, null, null],
-[null, null, <ore:plateDoubleDraconium>, <gregtech:gt.metaitem.01:32744>, <EnderIO:blockCapBank:1>.withTag({type: "SIMPLE"}), <gregtech:gt.blockmachines:263>, <ore:plateDoubleDraconium>, null, null],
-[null, null, <ore:plateDoubleDraconium>, <ore:chestDiamond>, WyvernCore, <ore:chestDiamond>, <ore:plateDoubleDraconium>, null, null],
-[null, null, <ore:screwDraconium>, <ore:plateDoubleDraconium>, <ore:plateDoubleDraconium>, <ore:plateDoubleDraconium>, <ore:screwDraconium>, null, null],
-[null, null, null, null, null, null, null, null, null],
-[null, null, null, null, null, null, null, null, null],
-[null, null, null, null, null, null, null, null, null]]);
+recipes.addShaped(<DraconicEvolution:draconiumChest>, [
+[<ore:plateObsidian>, <EnderIO:blockCapBank:1>, <ore:plateObsidian>],
+[<IC2:blockMachine:1>, <avaritiaddons:CompressedChest>, <gregtech:gt.metaitem.01:32744>],
```

Comment:
<IC2:blockMachine:1> is an iron furnace? why?

Reply:
Because it has a furnace in the chest?

Reply:
ah ok the its fine